src/platform/K32W/Logging.cpp

```diff
@@ -21,60 +24,56 @@
 static bool isLogInitialized;
 extern uint8_t gOtLogUartInstance;
 extern "C" void K32WWriteBlocking(const uint8_t * aBuf, uint32_t len);
+extern "C" uint32_t otPlatAlarmMilliGetNow(void);
 
 namespace chip {
 namespace Logging {
 namespace Platform {
 
-void GetMessageString(char * buf, uint8_t chipCategory, uint8_t otLevelLog)
+/* bufLen must have at least 4 (miliseconds timestamp) + 1 (category) +  */
+void GetMessageString(char * buf, uint8_t bufLen, const char * module, uint8_t category)
 {
-    if (chipCategory != kLogCategory_None)
+    int writtenLen = 0;
+    const char * categoryString;
+
+    assert(bufLen >= (TIMESTAMP_MAX_LEN_BYTES + CATEGORY_MAX_LEN_BYTES + strlen(module) + 1));
+
+    writtenLen = snprintf(buf, bufLen, "[%lu]", otPlatAlarmMilliGetNow());
```

Comment:
So this seems like it would use at most 12 chars, right?

Reply:
Right, opened #8451